tutorials/cifar10-distributed.ipynb

```diff
@@ -0,0 +1,1534 @@
+   ],
+   "metadata": {
+    "id": "KXpew9g3PUKI"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "source": [
+    "def log_basic_info(logger, config):\n",
+    "    logger.info(f\"Train on CIFAR10\")\n",
+    "    logger.info(f\"- PyTorch version: {torch.__version__}\")\n",
+    "    logger.info(f\"- Ignite version: {ignite.__version__}\")\n",
+    "    if torch.cuda.is_available():\n",
+    "        # explicitly import cudnn as torch.backends.cudnn can not be pickled with hvd spawning procs\n",
+    "        from torch.backends import cudnn\n",
+    "\n",
+    "        logger.info(\n",
+    "            f\"- GPU Device: {torch.cuda.get_device_name(idist.get_local_rank())}\"\n",
+    "        )\n",
+    "        logger.info(f\"- CUDA version: {torch.version.cuda}\")\n",
+    "        logger.info(f\"- CUDNN version: {cudnn.version()}\")\n",
+    "\n",
+    "    logger.info(\"\\n\")\n",
+    "    logger.info(\"Configuration:\")\n",
+    "    for key, value in config.items():\n",
+    "        logger.info(f\"\\t{key}: {value}\")\n",
+    "    logger.info(\"\\n\")\n",
+    "\n",
+    "    if idist.get_world_size() > 1:\n",
+    "        logger.info(\"\\nDistributed setting:\")\n",
+    "        logger.info(f\"\\tbackend: {idist.backend()}\")\n",
+    "        logger.info(f\"\\tworld size: {idist.get_world_size()}\")\n",
+    "        logger.info(\"\\n\")"
+   ],
+   "outputs": [],
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-09-14T17:01:22.883100Z",
+     "iopub.status.busy": "2021-09-14T17:01:22.882657Z",
+     "iopub.status.idle": "2021-09-14T17:01:22.891892Z",
+     "shell.execute_reply": "2021-09-14T17:01:22.891225Z",
+     "shell.execute_reply.started": "2021-09-14T17:01:22.883044Z"
+    },
+    "id": "g61dzVvnEVvG"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This is a standard utility function to log `train` and `val` metrics after `validate_every` epochs."
+   ],
+   "metadata": {
+    "id": "5QT7tiAcJEk0"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "source": [
+    "def log_metrics(logger, epoch, elapsed, tag, metrics):\n",
+    "    metrics_output = \"\\n\".join([f\"\\t{k}: {v}\" for k, v in metrics.items()])\n",
+    "    logger.info(\n",
+    "        f\"\\nEpoch {epoch} - Evaluation time (seconds): {elapsed:.2f} - {tag} metrics:\\n {metrics_output}\"\n",
+    "    )"
+   ],
+   "outputs": [],
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-09-14T17:01:23.571959Z",
+     "iopub.status.busy": "2021-09-14T17:01:23.571522Z",
+     "iopub.status.idle": "2021-09-14T17:01:23.578820Z",
+     "shell.execute_reply": "2021-09-14T17:01:23.577344Z",
+     "shell.execute_reply.started": "2021-09-14T17:01:23.571919Z"
+    },
+    "id": "9hqmFjOJN8kK"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This is where the main logic resides, i.e. we will call all the above functions from within here:\n",
```

Comment:
@Priyansi please create another title here as switching from Logging to this phrase is not smooth and a bit confusing.